.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # Maintained by the MongoDB CLI team
-*       @mongodb/mongocli
+*       @mongodb/apix-2
```

Comment:
backport this change from master, will be updated to EA team after handover and code moved to separate repo